tests/test_roman.py

```diff
@@ -1052,6 +1052,84 @@ def test_config_sca():
         im2 -= sky_image
         assert im1 == im2
 
+@timer
+def test_aberration_interpolation():
+    
+    """Test the Roman aberration interpolation method inside roman.roman_psfs
+    """
+    # We read in pairs of conjunction points, they are on different SCAs but are physically
+    # adjacent on the FPA. The optical aberration between the two points in a pair should
+    # be much less than the aberration range in the FPA. The maximum and minimum of aberration
+    # in the FPA is pre-calculated using an 20x20 grid of positions on each SCA by Tianqing 
+    # Zhang. The conjunction pairs between the first row and the second row are considered 
+    # 'far', because are further separated. For 'far' pairs, z_diff<= 0.1*(z_max - z_min), 
+    # elsewhere, z_diff<= 0.05*(z_max - z_min).
+
+    print("Start continuity test for aberration interpolation")
+    
+    print('Read the SCA# and sca_pos of the conjunction pairs. (SCA#1, SCA1_X, SCA1_Y, SCA#2, SCA2_X, SCA2_Y )')
+    edge_pair_array = np.loadtxt("roman_files/sca_conjunction_points.txt")
+    # Z_min, Z_max across the FPA
+    Z_min_max = np.loadtxt('roman_files/Z_min_max.txt')
```

Comment:
I think this would be clearer to just calculate it from the aberration file directly after we read them in.

I think something like this should work:
```
Z_min = np.min(aberration_list, axis=(0,1))
Z_max = np.max(aberration_list, axis=(0,1))
```
but really, however you calculated the values to create this file is probably right.